Drogon failure on Azure, JOB_QUEUE_STATUS_FAILURE

**Describe the bug**
ERT fails on Drogon on Azure s034

**To reproduce**
Download Drogon
Activate `komodo 2022.12.latest`
Ensure `/global/distbin` is in your PATH
Modify the queue options in `drogon_design.ert` to include
`QUEUE_SYSTEM TORQUE`
`QUEUE_OPTION TORQUE DEBUG_OUTPUT torque_log.txt`
Run ensemble experiment, with or without the GUI

ERT will fail hard with "All realizations failed" and your terminal will give errors on:
`AssertionError: Unexpected job status type after running job: JOB_QUEUE_STATUS_FAILURE`
`Warning: failed to get job status for job:2600 from file:/tmp/ert-qstat-std-32307-512333`

In `torque_log.txt` you may find the errors:
```
qstat failed for job 2597, retrying in 4 seconds
qstat failed for job 2599, retrying in 4 seconds
qstat failed for job 2591, retrying in 4 seconds
qstat failed for job 2589, retrying in 4 seconds
qstat failed for job 2594, retrying in 4 seconds
qstat failed for job 2600, retrying in 4 seconds
qstat failed for job 2596, retrying in 8 seconds
qstat failed for job 2590, retrying in 8 seconds
qstat failed for job 2585, retrying in 8 seconds
qstat failed for job 2592, retrying in 8 seconds
qstat failed for job 2588, retrying in 8 seconds
qstat failed for job 2593, retrying in 8 seconds
qstat failed for job 2587, retrying in 8 seconds
qstat failed for job 2595, retrying in 8 seconds
qstat failed for job 2598, retrying in 8 seconds
qstat failed for job 2586, retrying in 8 seconds
qstat failed for job 2597, retrying in 8 seconds
qstat failed for job 2599, retrying in 8 seconds
qstat failed for job 2591, retrying in 8 seconds
qstat failed for job 2589, retrying in 8 seconds
qstat failed for job 2594, retrying in 8 seconds
qstat failed for job 2600, retrying in 8 seconds
```

and you can see that the file `/tmp/<yourusername>-ert-qstat` file being empty (this is the output from the proxy, which is in use)

When you get get to the terminal, no error is apparent, because the realization is still running, and `qstat 2600` gives a normal output.

**Expected behaviour**
No error.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment**
 - OS: RHEL7
 - ERT/Komodo release: 2022.12.rc9
 - Python version: 3.8
 - Remote/HPC execution involved: yes

--- a/src/clib/lib/job_queue/torque_driver.cpp
+++ b/src/clib/lib/job_queue/torque_driver.cpp
@@ -3,6 +3,7 @@
 #include <array>
 #include <fstream>
 #include <iostream>
+#include <math.h>
 #include <sstream>
 #include <stdio.h>
 #include <string.h>
@@ -442,8 +443,9 @@ static int torque_driver_submit_shell_job(torque_driver_type *driver,
 ￼                  retry a couple of times with exponential sleep.  */
                 int return_value = -1;
                 int sleep_time = 2;
-                int max_sleep_time = 2 * 2 * 2 * 2; /* max 4 attempts */
-                while ((return_value != 0) & (sleep_time < max_sleep_time)) {
+                int max_sleep_time = pow(sleep_time, 5);
+                // This gives a maximum cumulative sleep of 2+4+8+16+32 = 62 seconds
+                while ((return_value != 0) & (sleep_time <= max_sleep_time)) {
                     return_value = util_spawn_blocking(
                         driver->qsub_cmd, stringlist_get_size(remote_argv),
                         (const char **)argv, tmp_std_file, tmp_err_file);
@@ -551,9 +553,10 @@ torque_driver_get_qstat_status(torque_driver_type *driver,
            every second for every realization, thus the initial sleep time
            is 2 seconds. */
         int return_value = -1;
-        int sleep_time = 2;                 /* seconds */
-        int max_sleep_time = 2 * 2 * 2 * 2; /* max 4 attempts */
-        while ((return_value != 0) && (sleep_time < max_sleep_time)) {
+        int sleep_time = 2; /* seconds */
+        int max_sleep_time = pow(sleep_time, 5);
+        // This gives a maximum cumulative sleep of 2+4+8+16+32 = 62 seconds
+        while ((return_value != 0) && (sleep_time <= max_sleep_time)) {
             return_value =
                 util_spawn_blocking(driver->qstat_cmd, argv.size(), argv.data(),
                                     tmp_std_file, tmp_err_file);

